cmd/syncthing/externaladdr.go

```diff
@@ -67,8 +67,6 @@ func (e *addressLister) ExternalAddresses() []string {
 		}
 	}
 
-	l.Infoln("External addresses:", addrs)
```

Comment:
It's very spammy, it gets printed every time we do a local or global announce. We can still print it once at startup through, if you like.